include/tst_safe_pthread.h

```diff
@@ -18,6 +18,8 @@
 #ifndef TST_SAFE_PTHREAD_H__
 #define TST_SAFE_PTHREAD_H__
 
+#include <pthread.h>
+
 int safe_pthread_create(const char *file, const int lineno,
 			pthread_t *thread_id, const pthread_attr_t *attr,
 			void *(*thread_fn)(void *), void *arg);
```

Comment:
It's included in the tst_test.h at the moment, we should start with removing tst_safe_pthread.h from there and then adding it to a few tests that actually use it.